mappings/net/minecraft/util/NonNullList.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS fj net/minecraft/util/NonNullList
+	FIELD a delegate Ljava/util/List;
+	FIELD b defaultValue Ljava/lang/Object;
+	METHOD <init> (Ljava/util/List;Ljava/lang/Object;)V
+		ARG 1 delegate
+		ARG 2 defaultValue
+	METHOD a of ()Lfj;
+	METHOD a ofSize (ILjava/lang/Object;)Lfj;
```

Comment:
`ofFixedSize` to emphasize the fixed size

Reply:
Personally, I would reserve this semantic to documentation, as this makes the constructor seem differing to the semantics of the array copying constructor, and both return fixed-size lists.

Reply:
If anything we could probably make the no-args constructor imply a resizable list, which implicitly implies other constructors are fixed-size. But I still think it is a semantic best reserved for documentation to avoid verbose naming.